build/makelib/golang.mk

```diff
@@ -183,6 +187,14 @@ $(GO_TOOLS_DIR)/glide:
 	@mv $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)/glide $(GO_TOOLS_DIR)/glide
 	@rm -r $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)
 
+$(GO_TOOLS_DIR)/golint:
```

Comment:
I think this needs to be `$(GO_TOOLS_DIR)/lint/$(GOHOSTOS)-$(GOHOSTARCH)/golint:`